docs/guides/integration-mobile-overview.md

```diff
@@ -58,6 +49,72 @@ For details, refer to one of the following guides:
+
+1. What do you want to use to retrieve and refresh the UID2 token? See [Generating, Storing, and Refreshing the UID2 Token](#generating-storing-and-refreshing-the-uid2-token).
+
+1. How do you want to use the UID2 tokens? See [Sending the Token to the Bidstream](#sending-the-token-to-the-bidstream).
+
+### Client-Side or Client-Server Integration?
+
+The options for integrating with UID2 using the UID2 mobile SDKs are summarized in the following table. Choose the <Link href="../ref-info/glossary-uid#gl-integration-approaches">integration approach</Link> that's best for you.
+
+| Scenario | Option | Integration Guide |
+| :--- | :--- | :--- |
+| You have access to <Link href="../ref-info/glossary-uid#gl-dii">DII</Link> (email address or phone number) on the client side/within the mobile app, and want to keep changes within your app only. | Client-side integration | [UID2 Client-Side Integration Guide for Mobile](integration-mobile-client-side.md) |
+| You have access to DII on the server side only and can do the necessary development to generate UID2 tokens server-side, or you are using a <Link href="../ref-info/glossary-uid#gl-private-operator">Private Operator</Link>. | Client-Server Integration | [UID2 Client-Server Integration Guide for Mobile](integration-mobile-client-server.md) |
+
+### Generating, Storing, and Refreshing the UID2 Token
+
+The following table summarizes the mobile integration options for managing the UID2 token, including generating, storing, and refreshing the token, with links to the documentation for each option.
+
+| UID2 Mobile Implementation Option | SDK Doc | Implementation Guide |
+| :--- | :---| :---|
+| UID2 Android SDK | [SDK for Android Reference Guide](../sdks/sdk-ref-android.md) | One of the following:<ul><li>[UID2 Client-Side Integration Guide for Mobile](../guides/integration-mobile-client-side.md)</li><li>[UID2 Client-Server Integration Guide for Mobile](../guides/integration-mobile-client-server.md)</li></ul> |
+| UID2 iOS SDK | [SDK for iOS Reference Guide](../sdks/sdk-ref-ios.md) | One of the following:<ul><li>[UID2 Client-Side Integration Guide for Mobile](../guides/integration-mobile-client-side.md)</li><li>[UID2 Client-Server Integration Guide for Mobile](../guides/integration-mobile-client-server.md)</li></ul> |
+
+### Sending the Token to the Bidstream
+
+There are many options for sending your UID2 tokens to the bidstream.
+
+The following table summarizes the options supported by UID2.
+
+| Scenario | Integration Guide |
+| :--- | :--- |
+| You want to use Prebid Mobile SDK and Prebid Server to send out ad requests from your Android or iOS app | One of the following:<ul><li>Client-side: [Optional: UID2 Prebid Mobile SDK Integration](../guides/integration-mobile-client-side.md#optional-uid2-prebid-mobile-sdk-integration)</li><li>Client-server: [Optional: UID2 Prebid Mobile SDK Integration](../guides/integration-mobile-client-server.md#optional-uid2-prebid-mobile-sdk-integration)</li></ul> |
```

Comment:
maybe put this at the bottom of this table because in Functionality SUmmary section, Prebid Mobile is last